webapp/src/components/Settings/NavSettings.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { makeStyles, createStyles, Theme, Typography } from '@material-ui/core';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+
+const useStyles = makeStyles((theme: Theme) => createStyles({
+    root: {
+      flex: '0 0 20%',
+      minWidth: '200px',
+      height: '100%',
+      overflow: 'auto',
+      borderRight: `4px solid ${theme.palette.primary.dark}`,
+      backgroundColor: theme.palette.primary.main
+    },
+    userInfos : {
+      width: '100%',
+      padding: '10px 20px',
```

Comment:
the combination "width: 100%" + "padding" makes the component overflow its container.
just remove "width: 100%", since it's inside a flex container it will already take the full width